cli/wizard/commands/list-squid-token/index.ts

```diff
@@ -115,69 +115,95 @@ export async function listSquidToken() {
   const tokenListPath = path.resolve(process.cwd(), ...relativePath);
 
   await spinner("Creating PR...", async () => {
-    await $`git checkout -b feat/add-${newTokenConfig.symbol}-token`;
+    await $`git checkout -b feat/add-${newTokenConfig.prettySymbol}-token`;
     await $`git add ${tokenListPath}`;
-    await $`git commit -m "feat: add ${newTokenConfig.symbol} token"`;
+    await $`git commit -m "feat: add ${newTokenConfig.prettySymbol} token"`;
     await $`git push -u origin HEAD`;
   });
 }
 
 export type InterchainTokenInfo = {
   tokenId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   isOriginToken: boolean;
   isRegistered: boolean;
   chainId: number;
   axelarChainId: string;
   chainName: string;
   wasDeployedByAccount?: boolean;
-  kind: "canonical" | "standardized";
+  kind: "canonical" | "interchain" | "customInterchain";
```

Comment:
```suggestion
  tokenType: "canonical" | "interchain" | "customInterchain";
```

Reply:
this is also called `kind` downstream. I'll change the interface so that's clear